docs/source/dev/performance.rst

```diff
@@ -1,16 +1,18 @@
 Performance-Profiling and Optimization
 ======================================
-The OpenFAST team has been engaged in performance-profiling and optimization
-work in an effort to improve the time-to-solution performance for the most
-computationally expensive use cases. This work is supported by Intel® through
-its designation of NREL as an
+A major focus of he OpenFAST team is performance-profiling and optimization
```

Comment:
Are you going to add in this section any of the profiling results you found regarding linearization within WEIS-OpenFAST?  E.g., where are the computational bottlenecks in the linearization process--when computing the Jacobians of specific modules or when combining the Jacobians of each model into full-system-level Jacobians?